<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>KEY:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="inBox_2">
            <property name="minimumSize">
             <size>
              <width>91</width>
              <height>31</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_5">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Value:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inValue">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="On_insert_2">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(16, 100,144);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>insert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_delete">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(16, 100,144);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_ExtractMax">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(16, 100,144);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>contains</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="On_print">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(16, 100,144);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>print</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QPushButton" name="On_clear_2">
          <property name="minimumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>221</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(16, 100,144);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>clear</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>35</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <widget class="QWidget" name="page">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QTextEdit" name="textEdit"/>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>600</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <widget class="QWidget" name="verticalLayoutWidget_2">
            <property name="geometry">
             <rect>
              <x>-1</x>
              <y>9</y>
              <width>621</width>
              <height>602</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QGraphicsView" name="graphicsView"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>600</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
